src/app/static/src/tests/components/downloadResults/uploadModal.test.ts

```diff
@@ -378,12 +375,23 @@ describe(`uploadModal `, () => {
     });
 
     it("can clear timer when download action is complete", async () => {
-        const store = createStore();
+        const store = createStore(fakeMetadata, {
+            summary: {complete: false, downloading: false } as any,
+            spectrum: {complete: false, downloading: false} as any
+        });
         const wrapper = mount(UploadModal, {store})
-        jest.useFakeTimers()
 
         await wrapper.setProps({open: true})
-        wrapper.vm.$store.state.downloadResults = completeDownloadResults
+
+        //Refresh adrUpload upload files to trigger watch, which populates uploadFilesToADR
+        store.state.adrUpload.uploadFiles= {outputZip: fakeMetadata.outputZip};
+
+        //Click ok to trigger population of uploadFilesPayload
+        const okBtn = wrapper.find(".modal button.btn-red");
+        expect(okBtn.element)
+        await okBtn.trigger("click");
+
+        wrapper.vm.$store.state.downloadResults.spectrum.complete = true;
```

Comment:
Just setting the complete property in the download results rather than replacing the whole thing. This is to prevent `mockUploadFilesToADR` being invoked twice because of both `spectrum` and `summary` changing..